test/adapters/file/read.file.spec.js

```diff
@@ -0,0 +1,304 @@
+'use strict';
```

Comment:
I think `read.spec.js` would be a better name for this file since it's already in the `file` directory.